fetch.bs

```diff
@@ -1526,6 +1531,10 @@ Unless stated otherwise, it is zero.
 which is "<code>basic</code>", "<code>cors</code>", or "<code>opaque</code>".
 Unless stated otherwise, it is "<code>basic</code>".
 
+<p>A <a for=/>request</a> has an associated <dfn export for=request id=stale-content-mode>stale content mode</dfn>,
+which is "<code>allow</code>", "<code>disallow</code>" or "<code>default</code>". Unless stated
+otherwise, it is "<code>default</code>".
```

Comment:
I'm not 100% sure, but I'm not sure this is needed. Can't the current values of "cache mode" handle this?

Reply:
No they can't. There is no "must-revalidate" mode in the cache-mode. Reload causes it to skip the cache.

Reply:
What do we think about adding a new cache mode for that?  Currently it says the 'stale content mode' change is only applied for 'default', and other cache mode says something about cache behavior too, so it feels the mode can be one of the cache mode?

Reply:
If the only difference between "no-cache" and a SWR update request is the SWR update request doesn't add the `Cache-Control: max-age: 0` header, should we just add a flag to prevent that step?

Reply:
I've added a "no-cache local only flag" and then I can use the no-cache mode. If you have a better name I can change it.